backend/geonature/core/gn_meta/mtd/mtd_utils.py

```diff
@@ -140,8 +140,12 @@ def sync_af(af):
             .on_conflict_do_nothing(index_elements=["unique_acquisition_framework_id"])
             .returning(TAcquisitionFramework)
         )
+    DB.session.execute(statement)
```

Comment:
```suggestion
    acquisition_framework = DB.session.scalars(statement)
```
Normalement, cela fonctionne